### PR TITLE
Pull request, FarkoMoot!

### PR DESCRIPTION
**Pull request by FarkoMoot**

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this pull request! There are some changes that need to be made in order to format this correctly. Also, do not delete another person's file. Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Hi there, the `mtb.md` file is looking good, but the `time_to_live.md` and `coding.md` should be removed from the pull request (though you should not delete the files. Can you update this pull request and just stage the `mtb.md` without staging the other two files? Thank you.

**Comment by FarkoMoot** [comment]
Sorry, i'm beginner in this world of contributions.

but... i going to work for collaboration this.

**Pull request closed without merging**
